todo_app/app.py

```diff
@@ -8,7 +9,30 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    sorted_items = sorted(items, key=lambda i: i['status'], reverse=True)
+    return render_template('index.html', items=sorted_items)
+
+
+@app.route('/addItem', methods=['POST'])
+def add_items():
+    new_item = request.form.get('todoitem')
+    session_items.add_item(new_item)
+    return index()
+
+
+@app.route('/completeItem', methods=['POST'])
+def complete_items():
+    position_id = request.form['complete']
+    session_items.mark_as_complete_item(position_id)
+    return index()
+
+
+@app.route('/removeItem', methods=['POST'])
+def remove_items():
```

Comment:
From a UX perspective, it's generally better to do deletes/updates based on an id or similar rather than a title/name, usually because the name isn't necessarily unique. Here for example, if I have a weekly todo with the same name each time, I'll delete both the new and the old todo when I try to remove only the old one